deploy/crd/configauditreports.crd.yaml

```diff
@@ -53,3 +53,4 @@ spec:
     listKind: ConfigAuditReportList
     shortNames:
       - configaudit
+      - configaudits      
```

Comment:
@pradeepnnv can you remove all the space at the end of it? Also, after removing it, can you run `./hack/update-starboard.yaml.sh`? This will update the files under `deploy/static`.

Reply:
Hi @josedonizetti . Updated the PR. Thx for your help.